src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
@wkurniawan07 and @thyageshm, the two of you can cross check to ensure:
a) dev docs (where we explain process) explains usage of god mode
b) code comments or dev docs (where we explain design) explains the design/implementation details needed to maintain god mode. For this kind of documentation, code comments preferred, but some high level info may have to be put in dev docs.